### DISCUSSION TITLE
[SOLVED] [but still hoping for a more complete solution] Accessibility: Dark (inverted) Windows in Cinnamon for visually impaired users

### DISCUSSION THREAD

**Discussion by drapeaub** [linuxmint/discussions#712] [Ideas]
I would like Cinnamon to have a dark windows option (color inversion in desktop Windows) 
to allow a visually impaired person to read texts on the screen.

I know 4 people who use inverted screens in Microsoft Windows and the way Cinnamon 
is currently setup, Cinnamon applications are unreadable to them. 

My mother who is in this group of four tells me that the white panes are just too bright for her.
She cannot see the black letters on a white background: the letters are shaped as blobs of grey. 
It hurts her eyes.
At the moment she uses Windows 10 with an inverted color scheme: background of the pane: black, 
letters: white, highlights: bright yellow and purple. 
I would like her to switch to Linux Mint before the Windows 10's end of life (October 2025)
as she cannot read black texts on a white background. 

My ex-colleague, Sabrina, computer scientist, has an eyesight of 10% with a focal distance of 15 cm.
She was always using 150% enlargement to read her computer's screen. 
She is able program and respond to customers. 
But she can only read panes on a black background on Windows.
Ten years ago, she tried Ubuntu's desktop and it was useless: just too bright.
It had not improved since.

Microsoft Windows has done it right for visually impaired users.
I would like those features replicated in Cinnamon. 

My mother already uses LibreOffice, Thunderbird and Firefox on Windows 10. 
I would like to see these applications with a black background on Linux Mint. 
The transition would be easy for her. 
She used Linux Mint 17 in the past. 
LibreOffice works in inverted colors on Windows.
(I even opened a ticket because the letters of the menu were black on black background. 
It is now fixed. The inverted colors in the Microsoft Windows's panes work properly for LibreOffice.)

I still have to decide if I will install LM22 or LMDE6 on her computer.
It all depends on the availability of applications for the platform such 
as Zoom, Anydesk, Bitwarden and Skype. 

Do you think you can implement this solution before October 2025, EOL date for Microsoft Windows 10?
I would appreciate it.

Bernard

**Comment by mtwebster** [comment]
There's a Cinnamon applet that allows compensation for color-blindness and also has inversion capabilities - I'm not sure if this would solve all problems but may be worth investigating (if you haven't already):
![image](https://github.com/user-attachments/assets/79f7595b-de88-402c-ae1c-b90fe81495ec)

There are plenty of dark themes but generally you can't force applications to respect them. This applet, on the other hand, would affect the entire screen.

**Reply by Jopp-gh** [reply]
"There are plenty of dark themes but generally you can't force applications to respect them."

Indeed, only the dark versions of Mint-Y orange and Mint-Y green apply dark to all applications.

Other pre–installed dark themes or community dark themes are all code-incomplete and don't apply dark themes to all applications.

**Comment by claudiux** [comment]
There is an extension that negates window on Super+I.
https://cinnamon-spices.linuxmint.com/extensions/view/78

**Comment by Jopp-gh** [comment]
You're welcome
Mint comes already with many pre-installed dark themes: use the dark version of Mint-Y green or Mint-Y orange.

Follow this tutorial to find out how to find dark themes in your settings:
https://itsfoss.com/install-themes-linux-mint/

**Comment by drapeaub** [comment]
Thanks for your leads. I will investigate in the coming two weeks. I will return in January 2025.

Bernard

**Comment by drapeaub** [comment]
I investigated and the color-blind filters does the job.
I select Hi-Contrast and can toggle between positive and negative desktop.
in Zoom I can leave the positive image, in Libreoffice I use the negative image. 
the toggle in the bar is handy. 
The menus are always positive. 
But the general effect is what I am looking for. 

I would still want to have an inversion button in the accessibility dialog box so the menus would be inverted. 
But as working solution I am happy with the result.

Thank you for the leads.
Bernard
